SimpCity/Game.cs

```diff
@@ -223,8 +223,15 @@ public class Game {
                 for (int x = 0; x < grid.Width; x++) {
                     try {
                         CityGridBuilding b = grid.Get(new CityGridPosition(x, y));
-                        buildingScores[b.Info.Type].Add(b.CalcScore(calcArchive));
+                        if (b != null)
+                        {
+                            buildingScores[b.Info.Type].Add(b.CalcScore(calcArchive));
                         calcArchive.calculated(b);
+                        }
+                        else
+                        {
+                            continue;
+                        }
```

Comment:
Can we have braces on the same line to keep consistent with the project formatting?
```cs
if (condition) {
...
} else {
   continue
}
```